x-pack/plugins/apm/public/store/reactReduxRequest/transactionOverviewCharts.tsx

```diff
@@ -69,3 +72,26 @@ export function TransactionOverviewChartsRequest({ urlParams, render }: Props) {
     />
   );
 }
+
+// Ignores transaction type from urlParams and requests charts
+// for ALL transaction types within this service
+export function TransactionOverviewChartsRequestForAllTypes({
+  urlParams,
+  render
+}: Props) {
+  const { serviceName, start, end, kuery } = urlParams;
+
+  if (!(serviceName && start && end)) {
+    return null;
+  }
+
+  return (
+    <Request
+      id={ID}
+      fn={loadOverviewChartsForAllTypes}
+      args={[{ serviceName, start, end, kuery }]}
+      selector={getTransactionOverviewCharts}
+      render={render}
+    />
+  );
+}
```

Comment:
Next iteration we can hopefully clean up the `transactionType` implicitness, and add the appropriate redirects as we talked about, and then get rid of this and just use `TransactionOverviewChartsRequest`.